### DISCUSSION TITLE
Is there a `yarn info` equivalent command?

### DISCUSSION THREAD

**Discussion by raulfdm** [pnpm/pnpm#4728] [Q&A] [answered]
One nice feature from yarn (IMHO) is the `info` command. It retrieves information from a specific package in the terminal:

```bash
yarn info react
```

For me the most use case is getting the version from a package. For that I can specify `versions` and it'll only show up the list of versions:

```bash
yarn info react versions
```

Is there an equivalent command in pnpm?

**Comment by zkochan** [comment, marked as answer]
This command is also available in npm. So just run `npm info react versions`.

`pnpm info react versions` will also work but it will just run npm CLI under the hood.
